src/components/panel/base-panel.styles.tsx

```diff
@@ -0,0 +1,44 @@
+import { Color } from "@lifesg/react-design-system/color";
+import { MediaQuery } from "@lifesg/react-design-system/media";
+import { Transition } from "@lifesg/react-design-system/transition";
+import styled from "styled-components";
+
+export const BasePanelWrapper = styled.div<{ $isCollapsed: boolean }>`
```

Comment:
introduce a styled interface instead
```suggestion
export const Wrapper = styled.div<IWrapperStyleProps>`
```